wire-ios-data-model/Source/ConversationSearch/SearchConversationsUseCaseProtocol.swift

```diff
@@ -0,0 +1,25 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+/// Determines if a user has a valid E2EI certificate on all clients.
+public protocol SearchConversationsUseCaseProtocol {
+    associatedtype ConversationContainer: SearchableConversationContainer
+
+    /// Returns `true` if all clients of the provided conversation's user have valid E2EI certificates.
+    func invoke(searchText: String) -> [ConversationContainer]
```

Comment:
The comment "Returns `true`" seems outdated

Reply:
Thanks, doc comment fixed.